package.json

```diff
@@ -136,9 +136,13 @@
 		"iconv-lite": "^0.4.19",
 		"image-size": "^0.6.2",
 		"imap": "^0.8.19",
+		"ip-range-check": "0.0.2",
+		"jquery": "^3.2.1",
+		"jschardet": "1.6.0",
 		"ip-range-check": "^0.0.2",
 		"jquery": "^3.3.1",
 		"jschardet": "^1.6.0",
+    "lamejs": "^1.2.0",
```

Comment:
Are you still using the import?

Reply:
I think the idea was to use this instead of including the file.  But per DM struggling to get the import to work in the worker.

So need to figure out how to serve the file out of the npm package

Reply:
Yes, that was the purpose but I am not able to serve it from `node_modules`
I have removed the dependency for now.